src/SourceBuild/content/build.proj

```diff
@@ -10,7 +10,12 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
 
   <Target Name="Build">
-    <Message Text="Build Environment: $(TargetArchitecture) $(Configuration) $(TargetOS) $(TargetRid)" Importance="high" />
+    <PropertyGroup>
+      <SourceBuildInfoText Condition="'$(DotNetBuildSourceOnly)' == 'true'">(source-build)</SourceBuildInfoText>
+      <SourceBuildInfoText Condition="'$(DotNetBuildSourceOnly)' != 'true'">(unified-build)</SourceBuildInfoText>
```

Comment:
I would go with 'source-only' and, maybe `non-source-only`, or perhaps avoid any text in the non-source-only case. Technically they're both UB.

Reply:
Got it. Unfortunately we named our non-source-only pipelines unified-build. Maybe we want to change that name at some point.

Reply:
Yeah we'll want to change it to just "dotnet-dotnet" probably.